pyimzml/ImzMLWriter.py

```diff
@@ -309,18 +299,18 @@ def main(argv):
     try:
         opts, args = getopt.getopt(argv,"hi:o:",["ifile=","ofile="])
     except getopt.GetoptError:
-        print 'test.py -i <inputfile> -o <outputfile>'
+        print('test.py -i <inputfile> -o <outputfile>')
```

Comment:
For a single argument it happens to work indeed, two or more would be parsed as a tuple in python2.